src/database/models/User.ts

```diff
@@ -23,6 +57,7 @@ const UserSchema: Schema<UserModel> = new Schema(
     providerAccountId: {
       type: String,
     },
+    isOnboarded: OnboardingSchema,
```

Comment:
isOnboarded should be boolean.
create a new key for onboarding or better simply put into User Schema directly no need for Onboarding